src/components/ViewRoutingSlip/FIlingTypeAutoComplete.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <transition-group name="slide-fade">
+    <div class="d-flex" key="action">
+      <v-autocomplete
+        filled
+        v-model="filingType"
+        :items="autoCompleteFilingTypes"
+        :loading="isLoading"
+        :search-input.sync="search"
+        @keyup="delayedSearch"
+        item-text="filingTypeCode.description"
+        item-value="filingTypeCode.description"
+        placeholder="Filing Type Name"
+        return-object
+        append-icon=""
+      >
+        <!-- hide-no-data -->
+        <template v-slot:item="{ item }">
+          <div class="rs-details">
+            <span>{{ item.corpTypeCode.description }}</span>
```

Comment:
Shouldn't this be other way around ? item.filingTypeCode.description - item.corpTypeCode.description

Reply:
good catch